bitnami/wordpress/templates/deployment.yaml

```
@@ -2,6 +2,7 @@ apiVersion: {{ include "common.capabilities.deployment.apiVersion" . }}
 kind: Deployment
 metadata:
   name: {{ include "common.names.fullname" . }}
+  namespace: {{ .Release.Namespace }}
```

```suggestion
  namespace: {{ .Release.Namespace | quote }}
```

This solves issues with some namespaces names